Documentation/community.rst

```diff
@@ -4,6 +4,67 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+Weekly Community Meeting
+========================
+
+The Cilium contributors gather every Wednesday at 8am PDT, 17:00 CEST, for a
+one-hour Zoom call open to everyone. During that time, we discuss:
+
+- the statuses of the next releases for each supported Cilium release
+- the current state of our CI: flakes being investigated and upcoming
+  changes
+- the development items for the next release
+- miscellaneous topics during the open session
+
+If you want to discuss something during the next meeting's open session,
+you can add it to `the meeting's Google doc
+<https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#>`_.
+The Zoom link to the meeting is available in the #development Slack
+channel and in `the meeting notes
+<https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#>`_.
+
+Slack
+=====
+
+Our Cilium & eBPF Slack is the main discussion space for the Cilium community.
+Click `here <https://cilium.herokuapp.com>`_ to request an invite. 
+
+Slack channels
+--------------
+
+==================== ====================================
+Name                 Purpose
+==================== ====================================
+#general             General user discussions & questions
+#hubble              Questions on Hubble
+#kubernetes          Kubernetes-specific questions
+#networkpolicy       Questions on network policies
+#release             Release announcements only
+==================== ====================================
+
+You can join the following channels if you are looking to contribute to
+Cilium:
+
+==================== ====================================
+Name                 Purpose
+==================== ====================================
+#development         Development discussions
+#git                 GitHub notifications
+#sig-*               SIG-specific discussions (see below)
+#testing             Testing and CI discussions
+==================== ====================================
+
+If you are interested in eBPF, then the following channels are for you:
+
+==================== ====================================================================
+Name                 Purpose
+==================== ====================================================================
+#ebpf                eBPF-specific questions
+#ebpf-lsm            Questions on BPF LSM
+#ebpf-news           Contributions to the `eBPF Updates <https://ebpf.io/blog>`_
+#libbpf-go           Questions on the `eBPF Go library <https://github.com/cilium/ebpf>`_
+==================== ====================================================================
+
```

Comment:
LGTM, looks like the commit being backported here was based on top of the changes in 63996d674e844da369e6408db4b90c836952d5ba so we probably should have cherry-picked that one too or skipped this one. 

The test failure is related, as we're missing this hunk from that patch which would quell the failure:
```
diff --git a/Documentation/spelling_wordlist.txt b/Documentation/spelling_wordlist.txt
index c40db8ca99f9..253cd8c6fc4d 100644
--- a/Documentation/spelling_wordlist.txt
+++ b/Documentation/spelling_wordlist.txt
@@ -559,6 +559,7 @@ lookup
 lookups
 loopback
 lrp
+lsm
 lspci
 luke
 lwt
@@ -616,6 +617,7 @@ netfilter
 netperf
 netsec
 netvsc
+networkpolicy
 newproto
 newprotoparser
 nfp
```

Reply:
dropping it 🤷

Reply:
For this kind of documentation I think we should somehow focus only on `latest` and stop backporting it. Meeting information is a singleton and backporting it into various versions of the cilium documentation is quite silly.

Perhaps we can fix up the docs to always point to `:latest` for a few of these community pages

Reply:
I like that idea actually yeah, seems simpler. Just have to figure out either part of branching steps to truncate these docs & link to the latest, or some build-time switch that figures out which snippets to include depending on whether the version is latest or a specific version. Maybe there's a tidy trick for the latter approach, basically if the target docs build version is a real release, include a file that just links to the latest docs.